sync_radar_data_globus.py

```diff
@@ -157,9 +155,10 @@ def synchronize(self):
         try:
             # You absolutely need the '~' in front of the root of the path for the patterns to work.
             # This isn't obvious from documentation.
-            listing_path = "~/chroot/sddata/{}/{}/{}/".format(self.data_type,
-                                                              self.sync_year,
-                                                              self.sync_month)
+            listing_path = "{}/{}/{}/{}/".format(self.mirror_root_dir,
+                                                 self.data_type,
+                                                 self.sync_year,
+                                                 self.sync_month)
             # The listing pattern is handled by the python globus sdk
             listing_pattern = "name:~*{}*".format(self.sync_pattern)
```

Comment:
REVIEW 15:  curious what happens if the syn_pattern includes the file type? like 201701*.fitacf ?

Reply:
Should work just fine